doc/man5/config.pod

```diff
@@ -277,9 +277,25 @@ available to the provider.
 This modules has the name B<alg_section> which points to a section containing
 algorithm commands.
 
-Currently the only algorithm command supported is B<fips_mode> whose
-value can only be the boolean string B<off>. If B<fips_mode> is set to B<on>,
-an error occurs as this library version is not FIPS capable.
+The supported algorithms commands are:
+
+=over 4
+
+=item B<default_properties>
+
+The value may be anything that is acceptable as property query string for
+EVP_set_default_properties().
+
+=item B<fips_mode> (deprecated)
```

Comment:
?? Deprecated when it is introduced ??

Reply:
This is present in 1.0.2, and is still documented in 1.1.0 and 1.1.1, so not very "introduced" here.
Considering that the whole "fips mode" thing really belongs with the old FOM and that we only preserve the notion for backward compatibility, I do think that deprecation is appropriate.

Reply:
Good point.  I'd really like to remove it though.

Maybe document that setting the default properties in any manner replaces any previously existing default properties.

Reply:
>  I'd really like to remove it though.

So would I.  Is that permissible or will people get utterly confused?

> Maybe document that setting the default properties in any manner replaces any previously existing default properties.

Good point